modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -634,6 +641,174 @@ public void testDeleteCustomObjectEntry() throws Exception {
+
+		_objectEntryLocalService.deleteObjectEntry(objectEntry3);
+
+		JSONObject importedObjectEntryJSONObject =
+			HTTPTestUtil.invokeToJSONObject(
+				objectEntryExportJSONObject.toString(),
+				_getSiteScopedObjectDefinitionEndpoint(
+					siteScopedObjectDefinition, TestPropsValues.getGroupId()),
+				Http.Method.POST);
+
+		Assert.assertEquals(
+			0,
+			importedObjectEntryJSONObject.getJSONObject(
+				"status"
+			).get(
+				"code"
+			));
+
+		JSONObject nestedObjectEntryJSONObject =
+			HTTPTestUtil.invokeToJSONObject(
+				null,
+				_getEndpointNestedObjectEntry(
+					siteScopedObjectDefinition,
+					importedObjectEntryJSONObject.getLong("id"),
+					objectRelationship2.getName(),
+					objectRelationship1.getName()),
+				Http.Method.GET);
+
+		JSONArray objectEntry1JSONArray =
+			nestedObjectEntryJSONObject.getJSONArray("items");
+
+		Assert.assertNotNull(objectEntry1JSONArray);
+
+		JSONObject objectEntry1JSONObject = objectEntry1JSONArray.getJSONObject(
+			0);
+
+		Assert.assertEquals(
+			_objectEntry1.getExternalReferenceCode(),
+			objectEntry1JSONObject.get("externalReferenceCode"));
+
+		JSONArray objectEntry2JSONArray = objectEntry1JSONObject.getJSONArray(
+			objectRelationship1.getName());
+
+		Assert.assertNotNull(objectEntry2JSONArray);
+
+		JSONObject objectEntry2JSONObject = objectEntry2JSONArray.getJSONObject(
+			0);
+
+		Assert.assertEquals(
+			_objectEntry2.getExternalReferenceCode(),
+			objectEntry2JSONObject.get("externalReferenceCode"));
+	}
+
+	@Test
+	@TestInfo("LPD-60419")
+	public void testExportAndImportRelatedObjectEntryWithDifferentScopeAcrossSites()
+		throws Exception {
+
+		ObjectDefinition siteScopedObjectDefinition =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, true, null,
+						RandomTestUtil.randomString(), _OBJECT_FIELD_NAME_1,
+						false)),
+				ObjectDefinitionConstants.SCOPE_SITE);
+
+		_objectDefinitions.add(siteScopedObjectDefinition);
+
+		ObjectEntry objectEntry2 = ObjectEntryTestUtil.addObjectEntry(
+			siteScopedObjectDefinition, _OBJECT_FIELD_NAME_1,
+			_OBJECT_FIELD_VALUE_2);
+
+		Group group = GroupTestUtil.addGroup();
+
+		ObjectEntry objectEntry3 = ObjectEntryTestUtil.addObjectEntry(
+			group.getGroupId(), siteScopedObjectDefinition,
+			HashMapBuilder.<String, Serializable>put(
+				_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_2
+			).build());
+
+		ObjectRelationship objectRelationship = _addObjectRelationship(
+			_objectDefinition1, siteScopedObjectDefinition,
+			_objectEntry1.getPrimaryKey(), objectEntry2.getPrimaryKey(),
+			ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		ObjectRelationshipTestUtil.relateObjectEntries(
+			_objectEntry1.getPrimaryKey(), objectEntry3.getPrimaryKey(),
+			objectRelationship, TestPropsValues.getUserId());
+
+		JSONObject objectEntryExportJSONObject =
```

Comment:
Could you please consolidate objectEntryExportJSONObject and importedObjectEntryJSONObject into a single objectEntryJSONObject variable, reassigning it as needed?